### PR TITLE
Add `cloud-sun-rain` and `cloud-moon-rain`

### PR DESCRIPTION
**Pull request by it-is-not**
image

### CODE REVIEW

File: icons/cloud-sun-rain.svg [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  stroke-linejoin="round"
+>
+  <path d="M12 2v2"/>
+  <path d="m5.22 5.22 1.42 1.42"/>
+  <path d="M20 12h2"/>
+  <path d="M15.97 12.5A4 4 0 0 0 9.5 8.88"/>
+  <path d="m17.36 6.64 1.42-1.42"/>
+  <path d="M15.54 22a3.4 3.4 0 0 0-1.88-6.2h-1.35a5.3 5.3 0 0 0-5.46-3.78 5.24 5.24 0 0 0-4.8 4.6 5.33 5.33 0 0 0 1.46 4.4"/>
+  <path d="M11.5 19v3"/>
+  <path d="M7.5 18v3"/>
```
Comment (karsa-mistmere) [inline icons/cloud-sun-rain.svg:19]: The NW and NE strokes are a bit off compared to the rest:
```suggestion
  <path d="M12 2v2" />
  <path d="m4.93 4.93L6.34 6.34" />
  <path d="M20 12h2" />
  <path d="M15.97 12.5A4 4 0 0 0 9.5 8.88" />
  <path d="M17.66 6.34L19.07 4.93" />
  <path d="M15.54 22a3.4 3.4 0 0 0-1.88-6.2h-1.35a5.3 5.3 0 0 0-5.46-3.78 5.24 5.24 0 0 0-4.8 4.6 5.33 5.33 0 0 0 1.46 4.4" />
  <path d="M11.5 19v3" />
  <path d="M7.5 18v3" />
```

**Review by ericfennis — APPROVED** [review]
The drops looks a bit weird to me. But I see there is no room to move them a bit to the bottom. So it ok for now. Maybe can take a look later

**Comment by ericfennis** [comment]
@it-is-not I will say the first one.

**Review by ericfennis — CHANGES_REQUESTED** [review]
@it-is-not Can you update the icon?

**Review by ericfennis — APPROVED** [review]

**Pull request merged**